lib/broadway_dashboard.ex

```diff
@@ -154,22 +143,14 @@ defmodule BroadwayDashboard do
     else
       items =
         for name <- assigns.pipelines do
-          {name,
-           name: format_nav_name(name),
-           render: fn -> render_pipeline_or_error(assigns) end,
-           method: :redirect}
+          name = inspect(name)
+          {name, name: name, render: fn -> render_pipeline_or_error(assigns) end}
```

Comment:
I was just testing this, if there are multiple pipelines, without the :redirect the pipeline graph does not render when clicking the associated pipeline in the nav.

Reply:
@pringlewood thank you for testing this! 

@feng19 I think the `method: :redirect` is needed because we do a lot of things in the `mount/3` callback, specially the `Metrics.listen/3` part.

Reply:
Sorry, My mistake.

Reply:
No problem :)